index.bs

```diff
@@ -1037,8 +1045,16 @@ sequence&lt;{{WebTransportHash}}&gt; |serverCertificateHashes|, run these steps.
        protocol if present, following [[!WEB-TRANSPORT-OVERVIEW]]
        [Section 3.1](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-overview-09/#section-3.1),
        or `""` if not present.
-    1. If the connection is an HTTP/3 connection, set |transport|.{{[[Reliability]]}} to `"supports-unreliable"`.
-    1. If the connection is an HTTP/2 connection [[!WEB-TRANSPORT-HTTP2]], set |transport|'s {{[[Reliability]]}} to `"reliable-only"`.
+    1. If the connection is an HTTP/3 connection, run the following substeps:
```

Comment:
Should we link to the WebTransport over HTTP/3 document here, like we do for HTTP/2 below?